src/Faker/Provider/id_ID/Company.php

```diff
@@ -14,7 +14,7 @@ class Company extends \Faker\Provider\Company
     /**
      * @see http://id.wikipedia.org/wiki/Jenis_badan_usaha
      */
-    protected static $companyPrefix = ['PT', 'CV', 'UD', 'PD', 'Perum'];
+    protected static $companyPrefix = ['PT', 'Fa', 'CV', 'UD', 'PJ', 'PD', 'Perum', 'Yayasan'];
```

Comment:
Do we have reference docs for this?

Reply:
http://id.wikipedia.org/wiki/Jenis_badan_usaha same references